src/pages/whats-new.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { Container, Row, Col, ListGroup } from 'react-bootstrap';
+import { Link, Element } from "react-scroll";
+
+import twitterIcon from '../images/icons/twitter-icon-blue.svg'
+import NewFeature from "../components/new-feature"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+import newFeaturesData from "../../data/new-features.json"
+import image from "../images/feature-june.gif"
+
+import "../styles/pages/whats-new.scss"
+
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
```

Comment:
is this basically formatting the date into an id that `react scroll can reference`?

Reply:
yes!